lib/zold/remotes.rb

```diff
@@ -242,9 +244,16 @@ def if_present(host, port)
       save(list)
     end
 
+    def read_mtime
+      CSV.read(file, skip_lines: /.+,\d+,\d+,\d+/).map do |r|
```

Comment:
@emilianodellacasa `r` is probably better off as `row`

Reply:
@ledestin You are right, it is much more clear